test/test_line.py

```diff
@@ -905,6 +905,22 @@ def test_slope_getter(self):
             else:
                 self.assertEqual(l[0].slope, l[1])
 
+    def test_meth_at(self):
+        invalid_types = (None, [], "1", (1,), [1, 2, 3], Vector2(1, 1))
+
+        line = Line(0, 0, 1, 1)
+
+        # Test x1
+        for value in invalid_types:
+            with self.assertRaises(TypeError):
+                line.at(value)
+
+        self.assertEqual(line.at(0.5), (0.5, 0.5))
+        self.assertEqual(line.at(1), (1, 1))
+        self.assertEqual(line.at(0), (0, 0))
+        self.assertEqual(line.at(-1), (-1, -1))
+        self.assertEqual(line.at(2), (2, 2))
+
```

Comment:
You should add some checks that ensure that the function always returns a Tuple type